src/commands/integrations/flows/listen.test.ts

```diff
@@ -1,13 +1,189 @@
-import { describe, it, expect } from "bun:test";
+import { describe, it, expect, beforeAll, afterAll, afterEach, vi } from "vitest";
+import { encode } from "@msgpack/msgpack";
+import { setupServer } from "msw/node";
+import { graphql, HttpResponse, http } from "msw";
 import ListenCommand, { getTriggerType } from "./listen.js";
+import { ActionScheduleSupport } from "../../../graphql/schema.generated.js";
+import type { GetIntegrationFlowsQuery } from "../../../graphql/integrations/getIntegrationFlows.generated.js";
+import type { GetExecutionsQuery } from "../../../graphql/executions/getExecutions.generated.js";
+import type { GetPolledExecutionQuery } from "../../../graphql/executions/getPolledExecution.generated.js";
+import type { TestIntegrationFlowMutation } from "../../../graphql/integrations/testIntegrationFlow.generated.js";
+import type { UpdateIntegrationFlowListeningModeMutation } from "../../../graphql/integrations/updateIntegrationFlowListeningMode.generated.js";
+import { fs } from "../../../fs.js";
+import inquirer from "inquirer";
+
+vi.mock("../../../auth.js", () => ({
+  getAccessToken: vi.fn(() => Promise.resolve("test-token")),
+  prismaticUrl: "https://test.prismatic.io",
+}));
+
+vi.mock("../../../utils/http.js", async () => {
+  return {
+    fetch: (...args: Parameters<typeof fetch>) => fetch(...args),
+    createFetch: () => fetch,
+  };
+});
+
+vi.mock("../../../fs.js", () => ({
+  exists: vi.fn(() => Promise.resolve(true)),
+  fs: {
+    mkdir: vi.fn(() => Promise.resolve()),
+    writeFile: vi.fn(() => Promise.resolve()),
+  },
+}));
+
+vi.mock("inquirer", () => ({
+  default: {
+    prompt: vi.fn(() => Promise.resolve({ confirm: true, selectedFlow: null })),
+  },
+}));
+
+vi.mock("@oclif/core", async (importOriginal) => {
+  const actual = await importOriginal<typeof import("@oclif/core")>();
```

Comment:
More nitty but wrapping the first  arg in `import` should enable types to flow through (including to `importOriginal`).